mir_dataset_loaders/salami.py

```diff
@@ -0,0 +1,210 @@
+                          'title': None, 'artist': None, 'annotator_1_time': None, 'annotator_2_time': None,
+                          'class': None, 'genre': None}
+
+    annotations_dir = os.path.join(data_home, SALAMI_DIR, 'salami-data-public-master', 'annotations')
+    annotators = [any(SALAMI_INDEX[track_id]['annotator_1_uppercase']),
+                  any(SALAMI_INDEX[track_id]['annotator_2_uppercase'])]
+    all_annotators_section_data = _load_sections(get_local_path(annotations_dir, track_id), annotators)
+
+    return SalamiTrack(
+        track_id,
+        all_annotators_section_data[0],
+        all_annotators_section_data[1],
+        all_annotators_section_data[2],
+        all_annotators_section_data[3],
+        track_metadata['source'],
+        track_metadata['annotator_1_id'],
+        track_metadata['annotator_2_id'],
+        track_metadata['duration_sec'],
+        track_metadata['title'],
+        track_metadata['artist'],
+        track_metadata['annotator_1_time'],
+        track_metadata['annotator_2_time'],
+        track_metadata['class'],
+        track_metadata['genre']
+    )
+
+
+def _load_sections(sections_path, annotators):
+    all_annotators_section_data = []
+    for a in range(len(annotators)):
+        for f in ['uppercase.txt', 'lowercase.txt']:
+            times, secs = [], []
+            if annotators[a]:
+                file_path = os.path.join(sections_path, 'parsed',
+                                         'textfile{}_{}'.format(str(a + 1), f))
+                if os.path.exists(file_path):
+
+                    with open(file_path, 'r') as fhandle:
+                            reader = csv.reader(fhandle, delimiter='\t')
+                            for line in reader:
+                                times.append(float(line[0]))
+                                secs.append(line[1])
+                    times, secs = np.array(times), np.array(secs)
+                    # remove sections with length == 0
+                    times_ = np.delete(times, np.where(np.diff(times) == 0))
+                    secs = np.delete(secs, np.where(np.diff(times) == 0))
+                    all_annotators_section_data.append(SectionData(np.array(times_[:-1]),
+                                                       np.array(times_)[1:], np.array(secs)[:-1]))
+                else:
+                    all_annotators_section_data.append(None)
+            else:
+                all_annotators_section_data.append(None)
+
+    return all_annotators_section_data
+
+
+def _load_metadata(data_home):
+
+    metadata_relative_path = get_local_path(data_home, os.path.join(SALAMI_DIR,
+                                                                    'salami-data-public-master',
+                                                                    'metadata', 'metadata.csv'))
+    metadata_path = get_local_path(
```

Comment:
I think there's a bug here - `metadata_relative_path` will start with data_home already. i'm guessing you don't need `metadata_path` at all, just `metadata_relative_path`

Reply:
indeed, they're the same path, removing one